### PR TITLE
Add Gitea Community Code of Conduct

### PR DESCRIPTION
**Pull request by wolfogre**
The Gitea Community Code of Conduct.

Authored by lunny. Slightly modified from [Go CoC](https://go.dev/conduct)

---

CC all TOCs:

@go-gitea/technical-oversight-committee 

And we welcome suggestions from everyone.

### CODE REVIEW

File: CODE_OF_CONDUCT.md
Comment (techknowlogick) [inline CODE_OF_CONDUCT.md:19]: Thanks for this PR, CI is failing on this list due to the amount of indentation. Could you update it to be indented 2 spaces instead of 4?

File: CODE_OF_CONDUCT.md
Comment (delvh) [inline CODE_OF_CONDUCT.md]: `Giteaer`?
Sounds weird, especially since there are three subsequent vocals, which always looks strange.

File: CODE_OF_CONDUCT.md
Comment (delvh) [inline CODE_OF_CONDUCT.md:96]: We have such an email address?
Since when?

File: CODE_OF_CONDUCT.md
Comment (delvh) [inline CODE_OF_CONDUCT.md]: Either
```suggestion
Reports should be directed to the Gitea Project Stewards at conduct@gitea.com. It is the Project Stewards’ duty to receive and address reported violations of the code of conduct. They will then work with a committee consisting of representatives from the technical oversight committee.
```
or
```suggestion
Reports should be directed to the Gitea Project Stewards at conduct@gitea.com. It is the Project Stewards’ duty to receive and address reported violations of the code of conduct. They will then work with a committee consisting of representatives from the technical-oversight-committee.
```

File: CODE_OF_CONDUCT.md
Comment (delvh) [inline CODE_OF_CONDUCT.md]: Hmm, I'm not quite sure about this sentence:
> We will notify the accused of the report and provide them an opportunity to discuss it before any action is taken.

Does that mean when we see someone spamming, we should first ask them not to do that instead of directly timeouting/kicking them?
I mean for example for Discord that someone just joined our server, wrote a spam message, and was subsequently kicked…

While we can follow the CoC in this case, I think that's a lot of (unnecessary) extra effort for such a fruitless endeavor. That won't work on most trolls/spam factories.

File: CODE_OF_CONDUCT.md
Comment (lunny) [reply CODE_OF_CONDUCT.md:96]: Not yet. But I think maybe we can create one.

File: CODE_OF_CONDUCT.md
Comment (lunny) [reply CODE_OF_CONDUCT.md]: I agree with you we don't need to do that in that situation. Please feel free to give a proposal.

File: CODE_OF_CONDUCT.md
Comment (lunny) [reply CODE_OF_CONDUCT.md]: I think so. Any suggestion?

File: CODE_OF_CONDUCT.md
Comment (delvh) [reply CODE_OF_CONDUCT.md:96]: Well… We should have one before this PR is merged :grin:

File: CODE_OF_CONDUCT.md
Comment (jolheiser) [reply CODE_OF_CONDUCT.md]: `Community values` or similar imo. In the original this was `Gopher` as that is the nickname for Go devs, however I don't think we have such a nickname, so here it is largely irrelevant.

Comment (delvh) [reply CODE_OF_CONDUCT.md]: :+1: for `Community values`.
```suggestion
## Community values
```

File: CODE_OF_CONDUCT.md
Comment (delvh) [reply CODE_OF_CONDUCT.md]:
```suggestion
We will investigate every complaint, but you may not receive a direct response. We will use our discretion in determining when and how to follow up on reported incidents, which may range from not taking action to permanent expulsion from the project and project-sponsored spaces. Under normal circumstances, we will notify the accused of the report and provide them an opportunity to discuss it before any action is taken. If there is a consensus between maintainers that such an endeavor would be useless (i.e. in case of an obvious spammer), we reserve the right to take action without notifying the accused first. The identity of the reporter will be omitted from the details of the report supplied to the accused. In potentially harmful situations, such as ongoing harassment or threats to anyone’s safety, we may take action without notice.
```
Something like that?

Comment (lunny) [reply CODE_OF_CONDUCT.md]: I'm OK. But leave this to other English native speakers.

**Comment by delvh** [comment]
Ping.
This is a PR that should be merged rather earlier than later.

**Comment by wolfogre** [comment]
Need more help from other English native speakers.

**Review by lunny — APPROVED** [review]

File: CODE_OF_CONDUCT.md
Comment (kousu) [inline CODE_OF_CONDUCT.md]: I saw you needed help from native english speakers on this. This would read clearer to me with some punctuation: 

```suggestion
Project maintainers have the right and responsibility to remove, edit, or reject: comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, as well as to ban (temporarily or permanently) any contributor for behaviors that they deem inappropriate, threatening, offensive, or harmful.
```

**Comment by jolheiser** [comment]
For comparison, here is the diff of the current document (at time of comment) to the Go CoC.

<details>
<summary>Diff</summary>

```diff
diff --git a/go.md b/gitea.md
index 435bcc7..ef4151c 100644
--- a/go.md
+++ b/gitea.md
@@ -1,20 +1,20 @@
-# Go Community Code of Conduct
+# Gitea Community Code of Conduct
 
 ## About
 
-Online communities include people from many different backgrounds. The Go contributors are committed to providing a friendly, safe and welcoming environment for all, regardless of gender identity and expression, sexual orientation, disabilities, neurodiversity, physical appearance, body size, ethnicity, nationality, race, age, religion, or similar personal characteristics.
+Online communities include people from many different backgrounds. The Gitea contributors are committed to providing a friendly, safe and welcoming environment for all, regardless of gender identity and expression, sexual orientation, disabilities, neurodiversity, physical appearance, body size, ethnicity, nationality, race, age, religion, or similar personal characteristics.
 
-The first goal of the Code of Conduct is to specify a baseline standard of behavior so that people with different social values and communication styles can talk about Go effectively, productively, and respectfully.
+The first goal of the Code of Conduct is to specify a baseline standard of behavior so that people with different social values and communication styles can talk about Gitea effectively, productively, and respectfully.
 
 The second goal is to provide a mechanism for resolving conflicts in the community when they arise.
 
-The third goal of the Code of Conduct is to make our community welcoming to people from different backgrounds. Diversity is critical to the project; for Go to be successful, it needs contributors and users from all backgrounds. (See [Go, Open Source, Community](https://blog.golang.org/open-source).)
+The third goal of the Code of Conduct is to make our community welcoming to people from different backgrounds. Diversity is critical to the project; for Gitea to be successful, it needs contributors and users from all backgrounds.
 
 We believe that healthy debate and disagreement are essential to a healthy project and community. However, it is never ok to be disrespectful. We value diverse opinions, but we value respectful behavior more.
 
-## Gopher values
+## Community values
 
-These are the values to which people in the Go community (“Gophers”) should aspire.
+These are the values to which people in the Gitea community should aspire.
 
 - **Be friendly and welcoming.**
 - **Be patient.**
@@ -36,7 +36,7 @@ These are the values to which people in the Go community (“Gophers”) should
 - **Be responsible.**
   - What you say and do matters. Take responsibility for your words and actions, including their consequences, whether intended or otherwise.
 
-People are complicated. You should expect to be misunderstood and to misunderstand others; when this inevitably occurs, resist the urge to be defensive or assign blame. Try not to take offense where no offense was intended. Give people the benefit of the doubt. Even if the intent was to provoke, do not rise to it. It is the responsibility of _all parties_ to de-escalate conflict when it arises.
+People are complicated. You should expect to be misunderstood and to misunderstand others; when this inevitably occurs, resist the urge to be defensive or assign blame. Try not to take offense where no offense was intended. Give people the benefit of the doubt. Even if the intent was to provoke, do not rise to it. It is the responsibility of all parties to de-escalate conflict when it arises.
 
 ## Code of Conduct
 
@@ -80,17 +80,18 @@ We do not believe that all conflict is bad; healthy debate and disagreement ofte
 
 If you see someone violating the code of conduct, you are encouraged to address the behavior directly with those involved. Many issues can be resolved quickly and easily, and this gives people more control over the outcome of their dispute. If you are unable to resolve the matter for any reason, or if the behavior is threatening or harassing, report it. We are dedicated to providing an environment where participants feel welcome and safe.
 
-Reports should be directed to the Go Project Stewards at _conduct@golang.org_. It is the Project Stewards’ duty to receive and address reported violations of the code of conduct. They will then work with a committee consisting of representatives from the Open Source Programs Office and the Google Open Source Strategy team. If for any reason you are uncomfortable reaching out to the Project Stewards, please email the Google Open Source Programs Office at _opensource@google.com_.
+Reports should be directed to the Gitea Project Stewards at conduct@gitea.com. It is the Project Stewards’ duty to receive and address reported violations of the code of conduct. They will then work with a committee consisting of representatives from the technical-oversight-committee.
 
 We will investigate every complaint, but you may not receive a direct response. We will use our discretion in determining when and how to follow up on reported incidents, which may range from not taking action to permanent expulsion from the project and project-sponsored spaces. We will notify the accused of the report and provide them an opportunity to discuss it before any action is taken. The identity of the reporter will be omitted from the details of the report supplied to the accused. In potentially harmful situations, such as ongoing harassment or threats to anyone’s safety, we may take action without notice.
 
 ### Attribution
 
-This Code of Conduct is adapted from the Contributor Covenant, version 1.4, available at [https://www.contributor-covenant.org/version/1/4/code-of-conduct.html](https://www.contributor-covenant.org/version/1/4/code-of-conduct.html)
+This Code of Conduct is adapted from the Contributor Covenant, version 1.4, available at https://www.contributor-covenant.org/version/1/4/code-of-conduct.html
 
 ## Summary
 
 - Treat everyone with respect and kindness.
 - Be thoughtful in how you communicate.
 - Don’t be destructive or inflammatory.
-- If you encounter an issue, please mail [conduct@golang.org](mailto:conduct@golang.org).
+- If you encounter an issue, please mail conduct@gitea.com.
+
```
</details>

It's not much different aside from Go -> Gitea, which I think is fine considering the point is to work from a solid base.

One thing that may be important to note or act upon is the question of "what if someone has a problem with a TOC member?".  
The wording is a bit vague regarding who `conduct@` is composed of. Is it a new group?
In the Go CoC there is an alternative email given, however we don't exactly have the same form of governance. Perhaps we need wording that such reports can be made to any given TOC member, in case a report to `coduct@` would also include someone within that group?

**Comment by lunny** [comment]
> For comparison, here is the diff of the current document (at time of comment) to the Go CoC.
> Diff
> 
> ```diff
> diff --git a/go.md b/gitea.md
> index 435bcc7..ef4151c 100644
> --- a/go.md
> +++ b/gitea.md
> @@ -1,20 +1,20 @@
> -# Go Community Code of Conduct
> +# Gitea Community Code of Conduct
>  
>  ## About
>  
> -Online communities include people from many different backgrounds. The Go contributors are committed to providing a friendly, safe and welcoming environment for all, regardless of gender identity and expression, sexual orientation, disabilities, neurodiversity, physical appearance, body size, ethnicity, nationality, race, age, religion, or similar personal characteristics.
> +Online communities include people from many different backgrounds. The Gitea contributors are committed to providing a friendly, safe and welcoming environment for all, regardless of gender identity and expression, sexual orientation, disabilities, neurodiversity, physical appearance, body size, ethnicity, nationality, race, age, religion, or similar personal characteristics.
>  
> -The first goal of the Code of Conduct is to specify a baseline standard of behavior so that people with different social values and communication styles can talk about Go effectively, productively, and respectfully.
> +The first goal of the Code of Conduct is to specify a baseline standard of behavior so that people with different social values and communication styles can talk about Gitea effectively, productively, and respectfully.
>  
>  The second goal is to provide a mechanism for resolving conflicts in the community when they arise.
>  
> -The third goal of the Code of Conduct is to make our community welcoming to people from different backgrounds. Diversity is critical to the project; for Go to be successful, it needs contributors and users from all backgrounds. (See [Go, Open Source, Community](https://blog.golang.org/open-source).)
> +The third goal of the Code of Conduct is to make our community welcoming to people from different backgrounds. Diversity is critical to the project; for Gitea to be successful, it needs contributors and users from all backgrounds.
>  
>  We believe that healthy debate and disagreement are essential to a healthy project and community. However, it is never ok to be disrespectful. We value diverse opinions, but we value respectful behavior more.
>  
> -## Gopher values
> +## Community values
>  
> -These are the values to which people in the Go community (“Gophers”) should aspire.
> +These are the values to which people in the Gitea community should aspire.
>  
>  - **Be friendly and welcoming.**
>  - **Be patient.**
> @@ -36,7 +36,7 @@ These are the values to which people in the Go community (“Gophers”) should
>  - **Be responsible.**
>    - What you say and do matters. Take responsibility for your words and actions, including their consequences, whether intended or otherwise.
>  
> -People are complicated. You should expect to be misunderstood and to misunderstand others; when this inevitably occurs, resist the urge to be defensive or assign blame. Try not to take offense where no offense was intended. Give people the benefit of the doubt. Even if the intent was to provoke, do not rise to it. It is the responsibility of _all parties_ to de-escalate conflict when it arises.
> +People are complicated. You should expect to be misunderstood and to misunderstand others; when this inevitably occurs, resist the urge to be defensive or assign blame. Try not to take offense where no offense was intended. Give people the benefit of the doubt. Even if the intent was to provoke, do not rise to it. It is the responsibility of all parties to de-escalate conflict when it arises.
>  
>  ## Code of Conduct
>  
> @@ -80,17 +80,18 @@ We do not believe that all conflict is bad; healthy debate and disagreement ofte
>  
>  If you see someone violating the code of conduct, you are encouraged to address the behavior directly with those involved. Many issues can be resolved quickly and easily, and this gives people more control over the outcome of their dispute. If you are unable to resolve the matter for any reason, or if the behavior is threatening or harassing, report it. We are dedicated to providing an environment where participants feel welcome and safe.
>  
> -Reports should be directed to the Go Project Stewards at _conduct@golang.org_. It is the Project Stewards’ duty to receive and address reported violations of the code of conduct. They will then work with a committee consisting of representatives from the Open Source Programs Office and the Google Open Source Strategy team. If for any reason you are uncomfortable reaching out to the Project Stewards, please email the Google Open Source Programs Office at _opensource@google.com_.
> +Reports should be directed to the Gitea Project Stewards at conduct@gitea.com. It is the Project Stewards’ duty to receive and address reported violations of the code of conduct. They will then work with a committee consisting of representatives from the technical-oversight-committee.
>  
>  We will investigate every complaint, but you may not receive a direct response. We will use our discretion in determining when and how to follow up on reported incidents, which may range from not taking action to permanent expulsion from the project and project-sponsored spaces. We will notify the accused of the report and provide them an opportunity to discuss it before any action is taken. The identity of the reporter will be omitted from the details of the report supplied to the accused. In potentially harmful situations, such as ongoing harassment or threats to anyone’s safety, we may take action without notice.
>  
>  ### Attribution
>  
> -This Code of Conduct is adapted from the Contributor Covenant, version 1.4, available at [https://www.contributor-covenant.org/version/1/4/code-of-conduct.html](https://www.contributor-covenant.org/version/1/4/code-of-conduct.html)
> +This Code of Conduct is adapted from the Contributor Covenant, version 1.4, available at https://www.contributor-covenant.org/version/1/4/code-of-conduct.html
>  
>  ## Summary
>  
>  - Treat everyone with respect and kindness.
>  - Be thoughtful in how you communicate.
>  - Don’t be destructive or inflammatory.
> -- If you encounter an issue, please mail [conduct@golang.org](mailto:conduct@golang.org).
> +- If you encounter an issue, please mail conduct@gitea.com.
> +
> ```
> 
> It's not much different aside from Go -> Gitea, which I think is fine considering the point is to work from a solid base.
> 
> One thing that may be important to note or act upon is the question of "what if someone has a problem with a TOC member?". The wording is a bit vague regarding who `conduct@` is composed of. Is it a new group? In the Go CoC there is an alternative email given, however we don't exactly have the same form of governance. Perhaps we need wording that such reports can be made to any given TOC member, in case a report to `coduct@` would also include someone within that group?

The email address is a mail list will be forwarded to all members of TOC. I think it's OK to send an email to it so that every TOC member will receive it.

**Review by delvh — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

**Pull request merged**
